decidim-core/config/locales/en.yml

```diff
@@ -69,7 +69,8 @@ en:
     'true': 'Yes'
   carrierwave:
     errors:
-      image_too_big: The image is too big
+      file_resolution_too_large: File resolution is too large
+      file_too_large: File size is too large
```

Comment:
```suggestion
      file_size_too_large: File size is too large
```

I think it'd be nice to clarify it in the i18n key so it's more direct, what do you think?